tox.ini

```diff
@@ -4,28 +4,32 @@ args_are_paths = false
 envlist =
     flake8,
     isort,
-    py27-1.11,
-    py34-{1.11,2.0},
-    py35-{1.11,2.0,2.1,master},
-    py36-{1.11,2.0,2.1,master},
-    py37-{1.11,2.0,2.1,master}
+    py{27,36,37,38}-1.11,
+    py{36,37,38}-{2.0,2.1,2.2,master}
```

Comment:
2.0 and 2.1 are unsupported

2.2 supports 3.5 as well https://djangoci.com/job/django-2.2/